opds_import.py

```diff
@@ -607,10 +636,12 @@ def summary_to_linkdata(detail):
             data_source_last_updated=last_opds_update,
         )
             
-        # Only add circulation data if the data source is lendable.
-        if data_source.offers_licenses:
+        # Only add circulation data if both the book's distributor *and*
+        # the source of the OPDS feed are lendable data sources.
+        if (circulation_data_source and circulation_data_source.offers_licenses
+            and metadata_data_source.offers_licenses):
```

Comment:
This means that even if the metadata wrangler purports to offer licenses to books, we won't take it up on the offer.

I think this only happens in the tests, but it's good to fix the tests in a way that will stop a bad thing from happening in real life.